src/utils/__test__/randomizers.test.ts

```diff
@@ -0,0 +1,18 @@
+import assert from 'node:assert';
+import { describe, it } from 'node:test';
+
+import { randomEnumValue } from './randomizers.js';
+
+describe('randomEnumValue', () => {
+  it('should always return the value', () => {
+    /**
+     * Enums are compiled into a two-way mapping key → value *and* value → key
```

Comment:
I've modified the comment to explain this. I agree we should avoid enums because they are badly implemented in TS, but that's outside the scope of this PR.